### PR TITLE
Feature/mongodb jobmanager

### PR DESCRIPTION
**Pull request by xcomagent95**
# Overview
A jobmanager for the pygeoapi employing the mongoDB. 

# Related Issue / Discussion

#1179 

# Additional Information

# Contributions and Licensing

(as per https://github.com/geopython/pygeoapi/blob/master/CONTRIBUTING.md#contributions-and-licensing)

- [x] I'd like to contribute [feature X|bugfix Y|docs|something else] to pygeoapi. I confirm that my contributions to pygeoapi will be compatible with the pygeoapi license guidelines at the time of contribution.
- [ ] I have already previously agreed to the pygeoapi Contributions and Licensing Guidelines

### CODE REVIEW

**Review by tomkralidis — CHANGES_REQUESTED** [review]
Small changes suggested.

As well, add a section before https://github.com/geopython/pygeoapi/blob/master/docs/source/data-publishing/ogcapi-processes.rst#putting-it-all-together, called 'MongoDB', which provides 1-2 sentences on your plugin, as well as a sample configuration/example.

**Review by tomkralidis — CHANGES_REQUESTED** [review]
Thanks for the updates!  There are some flake8 errors that should be fixed (please ensure `flake8 pygeoapi/process/manager/mongodb_.py` runs without error).

As well, add a section before https://github.com/geopython/pygeoapi/blob/master/docs/source/data-publishing/ogcapi-processes.rst#putting-it-all-together, called 'MongoDB', which provides 1-2 sentences on your plugin, as well as a sample configuration/example.

**Comment by xcomagent95** [comment]
Hi,
I hope i could fix alle the issues mentioned.

File: pygeoapi-config.yml [hunk 1]
```diff
@@ -53,7 +53,6 @@ server:
 #        connection: /tmp/pygeoapi-process-manager.db
 #        output_dir: /tmp/
     # ogc_schemas_location: /opt/schemas.opengis.net
-
```
Comment (tomkralidis) [inline pygeoapi-config.yml:56]: Add this blank line back.

**Review by tomkralidis — CHANGES_REQUESTED** [review]

**Comment by tomkralidis** [comment]
FYI minor change request, and needs flake8 fixes. There are some flake8 errors that should be fixed (please ensure `flake8 pygeoapi/process/manager/mongodb_.py` runs without error).

**Pull request merged**